origami.py

```diff
@@ -75,10 +75,10 @@ class PaneCommand(sublime_plugin.WindowCommand):
 	"""Abstract base class for commands."""
 
 	def get_layout(self):
-		layout = self.window.layout()
-		cells = layout["cells"]
+		layout = self.window.get_layout()
```

Comment:
On master this has been switched to `self.window.layout()` so let's keep that.

Reply:
Good catch, I didn't notice that when resolving conflicts!

Reply:
And there is apparently one more place in the file that uses `window.get_layout()` which we can also update. `get_layout()` is apparently deprecated.